### ISSUE TITLE
Suggestion: Drawable

### ISSUE THREAD

**Issue by johnnybigoode** [HeapsIO/heaps-doc#41]
So I'm looking at H2D docs making some silly examples to understand what can be achieved. 

![2020-05-23_17-13-12](https://user-images.githubusercontent.com/2698584/82739772-e4b40380-9d18-11ea-8168-57833709d1be.gif)

The gif is just that I'm happy because this is the kind of engine I was looking when I tried Unity and Godot. 

That being said... The docs state that `Each Drawable (including h2d.Bitmap) has several properties that can be manipulated:` and among them I could use `blendMode`.

Welp, the properties are not from Drawable, they are from [h2d.Object](https://github.com/HeapsIO/heaps/blob/55dfef3be0ecd3ff6d79fc66695aa07e42ac1148/h2d/Object.hx). 

Erm, is this correct?

Sources:
* https://heaps.io/documentation/drawable.html

**Comment by ncannasse** [comment]
Yes it was moved there. But only works for Drawable or when the Object has
a filter.

Le sam. 23 mai 2020 à 22:21, johnnybigoode <notifications@github.com> a
écrit :
